site/docs/components/link.mdx

```diff
@@ -0,0 +1,129 @@
+- The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context.
+- Icons should always be the same colour as the link text
+
+## Usage and variations
+
+### Inline
+These links are the default link variant. Inline links are used within a sentence or paragraph and are underlined.
+
+The underlining helps differentiate them from the text they are placed next to and makes it clear users can interact with them. Inline links should not be used on their own.
+
+<Playground>
+  <p style={{ fontSize: '14px' }}>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+    magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+    <Link href="https://github.com/City-of-Helsinki/helsinki-design-system" external openInExternalDomainAriaLabel="Opens a different website">
+      HDS Github
+    </Link>
+    consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+    pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+    laborum.
+  </p>
+</Playground>
+
+#### Core:
+```html
+<p style={{ fontSize: '14px', lineHeight: '1.5' }}>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+  magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+  <a href="https://github.com/City-of-Helsinki/helsinki-design-system" class="hds-link hds-link--small" aria-label="HDS Github. Opens in external domain">
+  HDS Github<i class="hds-icon icon hds-icon--link-external hds-icon--size-xs vertical-align-small-or-medium-icon" aria-hidden="true"></i></a>
+  laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+  Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+</p>
+```
+
+#### React:
+```tsx
+<p style={{ fontSize: '14px' }}>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+  magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+  <Link href="https://github.com/City-of-Helsinki/helsinki-design-system" external openInExternalDomainAriaLabel="Opens a different website">
+    HDS Github
+  </Link>
+  consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+  pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+  laborum.
+</p>
+```
+
+### Standalone
+Standalone links are used on their own or directly following content, and they are underlined. If you wish not to have underline, consider using button instead.
```

Comment:
directly **in the** following

Reply:
If you wish not to have **an** underline, consider using **a** button instead.

Reply:
> directly **in the** following

- [ ] directly **in the** following

Reply:
> If you wish not to have **an** underline, consider using **a** button instead.

- [ ] If you wish not to have **an** underline, consider using **a** button instead.

Reply:
> "directly **in the** following"
> Or if it is meant to say after the content then:
> "Standalone links are used on their own or directly **after** the content, and they are underlined."

- [ ] Standalone links are used on their own or directly **after** the content, and they are underlined.